### DISCUSSION TITLE
Add support for freedesktop XDG_CONFIG_HOME

### DISCUSSION THREAD

**Discussion by jman-schief** [BlitterStudio/amiberry#1418] [Ideas]
Hello @midwan ,

I saw in commit c52b6c1acc0eeb4a5816c4da8505169c5c865783 you added suppot for `$XDG_DATA_DIR`. This is great!

Currently I support the XDG config dir with a little crutch like this (simplified):
```
#!/bin/env sh
config_dir=$XDG_CONFIG_HOME/amiberry
amiberry \
  -o path=$config_dir \
  -o config_path=$config_dir
```

Are there plans to also support natively `$XDG_CONFIG_HOME`? Would be great :slightly_smiling_face: 

Thanks again for Amiberry!

**Comment by midwan** [comment]
Yes, it's certainly possible to add that. The `$XDG_DATA_DIR` was originally added some time ago, to support the Flatpak installation.
If we add the `$XDG_CONFIG_HOME` one as well, I'll probably need to adjust a few things in the flatpak packaging, but otherwise should work fine.

**Reply by midwan** [reply]
@jman-schief this is now added, as mentioned in the related issue: #1425
Please test and let me know if it works as expected for you.

Note: the directories are currently _not_ created by Amiberry. It will search if they exist and use them on startup. This is something to improve in a future update (creating the directories if they don't exist, need to decide on the location that should happen, depending on the system).

**Reply by jman-schief** [reply]
yu-huu thank you! I'll download the next binary release
